### PR TITLE
Add "Empty" base map layer

### LINKED ISSUE #495: "Empty" basemap missing
We used to have a base layer called "Empty", but apparently that doesn't exist anymore.

### PR DESCRIPTION
**Pull request by Harry-R**
Adds "Empty" base map layer, fixes #495.

### CODE REVIEW

**Comment by Turbo87** [comment]
it seems that we get some sort of grey background when using the "Empty" basemap like this. would it be possible to use white instead?

**Comment by Harry-R** [comment]
The gray background seems to be the default skylines background, but i can try to change it for here.

**Comment by Turbo87** [comment]
hmm, yeah, that could be it. feel free to add a `background-color: white` style to the maps stylesheet.

File: ember/app/styles/_map.scss [hunk 1]
```diff
@@ -1,3 +1,7 @@
+.fullscreen-below-nav-bar {
```
Comment (Turbo87) [inline ember/app/styles/_map.scss]: can we use `.ol-viewport` here instead?

**Comment by Turbo87** [comment]
👍

**Pull request merged**